### ISSUE TITLE
Can't see prompt lines in install...?

### ISSUE THREAD

**Issue by mschendel** [dnschneid/crouton#2739]
```
Please paste the output of the following command here: sudo edit-chroot -all
```

#### Please describe your issue:
I do this command:
sudo sh ~/Downloads/crouton -t xfce
The last thing I see is 
ldconfig deferred processing now taking place
and then nothing. I've waited hours.  I tried killing it and rming the directory and starting the install over, and I tried killing it and trying crouton -u -t xfce, and it's done the same thing.

When I killed it, I noticed a weird thing.  The shell is like, laggy. and if I look, when I do ctrl C, I see something else return along with it: Please specify a username for the primary user:
When I tried restarting the install, I waited until I saw "ldconfig deferred processing now taking place" and I typed my username & enter. The shell advanced, and I saw this: 
Please specify a username for the primary user: my_username
When I hit enter again I see:
Enter new UNIX password: 
SO It seems like my shell isn't showing me the most recent line?  This makes "answer[ing] the prompts like a good person" very challenging....
I will try to do it by guessing what the current prompt line is showing, we'll see what happens.

I suppose I changed the chromebook's display resolution from default. Maybe that's it?

INFO:
chronos@localhost / $ cat /etc/lsb-release
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={BBE772C3-5541-6976-C676-54FD72D41B2B}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={BBE772C3-5541-6976-C676-54FD72D41B2B}
CHROMEOS_RELEASE_BOARD=kip-signed-mp-v2keys
CHROMEOS_RELEASE_BRANCH_NUMBER=68
CHROMEOS_RELEASE_BUILD_NUMBER=8350
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=52
CHROMEOS_RELEASE_DESCRIPTION=8350.68.0 (Official Build) stable-channel kip 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=8350.68.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=8350.68.0
 
#### If known, describe the steps to reproduce the issue:
sudo sh ~/Downloads/crouton -t xfce

**Comment by mschendel** [comment]
Huh.  After hitting random keys to see what the next prompts might be, it closed out ending with  
 /mnt/stateful_partition/crouton/chroots/precise...
I tried the install again with the update.  This time after 
ldconfig deferred processing now taking place
... it seems to have completed.  
You can start Xfce via the startxfce4 host command: sudo startxfce4

Unmounting /mnt/stateful_partition/crouton/chroots/precise...
Done! You can enter the chroot using enter-chroot.

Yay?  But now I have no primary user and no password for such?  Which I guess doesn't matter anyways, as this thing is currently not secure anyway?


Here is what I got the previous time, when it eventually failed:
Retype new UNIX password: 
Sorry, passwords do not match
passwd: Authentication token manipulation error
passwd: password unchanged
Enter new UNIX password: 
Retype new UNIX password: 
No password supplied
Enter new UNIX password: 
Retype new UNIX password: 
No password supplied
Enter new UNIX password: 
Retype new UNIX password: 
No password supplied
passwd: Authentication token manipulation error
passwd: password unchanged
Unmounting /mnt/stateful_partition/crouton/chroots/precise...

**Comment by DennisLfromGA** [comment]
@mschendel,

For security reasons, the password you enter will *not* be shown on the screen but it *is* being accepted, the system asks for it twice to be sure that what is entered each time is the same.

At this point though, you may be able to get into your chroot as root and set the password for the username you specified, try something like this:

  `sudo enter-chroot -u root`
  `passwd [username]` (you'll need to enter the password twice and it won't be displayed)
  `exit`

Be sure to replace `[username]` above with the username you specified when you installed your chroot.

----
If it were me, I would delete the semi-installed 'precise' chroot using: `sudo delete-chroot -n precise`
Then install a brand new 'trusty' chroot since it's an LTS release and is better supported.
You can do that using something like this:

  `sudo sh ~/Downloads/crouton -r trusty -t xfce`

Hope this helps,
-**DennisL**

**Comment by mschendel** [comment]
Hi, thanks for responding.

The problem was not that I couldn't see my password, it was that the most
recent line of the terminal wasn't showing, so I didn't even know it was
wanting input.  Just thought I would report that I was having this issue,
in case it happens to anyone else.  I think it might have been that I
didn't have my display set to the default?  Oh well.  I got through it.

It seems my blind button mashing eventually got me through the prompts, and
I was able to startxfce.  I followed your suggestions, and removed the
precise chroot and installed the trusty one.  So far so good!

Thanks,
Megan

MEG Technician
The Mind Research Network
1101 Yale Blvd. NE
Albuquerque, New Mexico 87106
505-272-3304
